// import{map, of, onErrorResumeNext} from 'rxjs'
// const source1=of('a','b','c', 'd')
// const source2= of('fallback')


// source1.pipe(map(value=>{
//     if(value==='c'){
//         throw new Error('An error occured')
//     }
//     return value
// }), 
// onErrorResumeNext())